src/Database/Expression/QueryExpression.php

```diff
@@ -725,6 +725,11 @@ protected function _addConditions(array $conditions, array $types): void
 
             $isArray = is_array($c);
             $isOperator = $isNot = false;
+            if($isArray && !$numericKey && count($c) === 2 && strpos(strtolower($k),' between ') !== false){
+                $parts = explode(' ',$k);
```

Comment:
I don't think this will work as field names can have spaces in them.

Reply:
That must have been the test that failed. Fair enough. You can deny the PR. I'll work towards migrating my app to utilize the new methodology. Thank you for your time!